src/FsAutoComplete.Suave/FsAutoComplete.Suave.fsproj

```diff
@@ -209,4 +186,5 @@
     </When>
   </Choose>
   <Import Project="..\..\packages\FSharp.Compiler.Service.ProjectCracker\build\$(__paket__FSharp_Compiler_Service_ProjectCracker_targets).targets" Condition="Exists('..\..\packages\FSharp.Compiler.Service.ProjectCracker\build\$(__paket__FSharp_Compiler_Service_ProjectCracker_targets).targets')" Label="Paket" />
+  <Import Project="..\..\packages\NETStandard.Library\build\NETStandard.Library.targets" Condition="Exists('..\..\packages\NETStandard.Library\build\NETStandard.Library.targets')" Label="Paket" />
```

Comment:
again, fishy

Reply:
This is because https://github.com/SuaveIO/suave/issues/638 – I'd love to move everything to a standard platform, but I can't even write F# right now, because my compilers and environment keeps crashing. Albeit due to this particular reference; but none the less, I'm in a bad position right now.

Reply:
@haf we are using suave 1.x here.
The issue is fixed here (thx to `paket why`) adding net45 restriction